src/main/java/teammates/common/util/Const.java

```diff
@@ -1403,6 +1403,9 @@ private Const() {
         public static final String STUDENT_COURSE_JOIN_SUCCESSFUL = "You have been successfully added to the course %s.";
         
         /** Template String. Parameters:  Course ID */
+        public static final String DELETED_STUDENT_ATTEMPTING_TO_JOIN =
+                "Unable to join course %s as you have been unenrolled by an instructor. Please contact your "
+                        + "course instructor for assistance.";
```

Comment:
The indentation here should be 8 spaces, i.e. `+ "course ...` should be aligned with `"Unable to ...`